bigchaindb/backend/query.py

```diff
@@ -141,6 +141,19 @@ def get_spent(connection, transaction_id, condition_id):
 
 
 @singledispatch
+def get_spending_transactions(connection, inputs):
+    """Return transactions which spend given inputs
+
+    Args:
+        inputs (list): list of {txid, output}
+
+    Returns:
+        List of transactions that spend given inputs
```

Comment:
Better to describe what is actually returned. In this case a tuple (block_id, transaction).
And the same for `get_owned_ids`